aws/resource_aws_route53_zone.go

```diff
@@ -389,6 +392,11 @@ func cleanZoneID(ID string) string {
 	return strings.TrimPrefix(ID, "/hostedzone/")
 }
 
+// cleanDomainName is used to remove the trailing period
+func cleanDomainName(v interface{}) string {
```

Comment:
Nit: Maybe name this new function to directly represent its functionality?

```suggestion
func trimTrailingPeriod(v interface{}) string {
```

Reply:
Ooo yeah much better 👍